jjb/go-modules/go-mod-messaging.yaml

```diff
@@ -5,15 +5,12 @@
     project-name: go-mod-messaging
     project: go-mod-messaging
     mvn-settings: go-mod-messaging-settings
-    build_script: 'make test'
-    stream:
-      - 'master':
-          branch: 'master'
-          pre_build_script: !include-raw-escape: shell/install_custom_golang.sh
-          go-root: '/opt/go-custom/go'
+    jenkins_file: 'Jenkinsfile'
+    status-context: '{project-name}-verify-pipeline'
+    stream: {}
```

Comment:
Most of this looks fine to me, but I'm curious about the removal of the stream. This removes the definition of `branch`, which in `build-images.yaml` is defined at the project level (see [this line](https://github.com/edgexfoundry/ci-management/blob/0c15ff97b17183ca06e5919ee5bb359e9e0ca7a4/jjb/build-images/build-images.yml#L7)). Should branch be defined in this job? And should we at least define an otherwise-empty stream?

Reply:
The overall thought is that we don't need separate jobs from different branches because the Jenkinsfile will define the different branch functionality in pipeline jobs. Collapsing all the verify and merge jobs into one job each. In most JJB templates stream is used for branches. In build-images we used it for the different images instead of branches. We have to at least define an empty object for stream or JJB will not parse it correctly and error out.